tests/Unit/Evolution/Systems/Cce/Test_WorldtubeData.cpp

```diff
@@ -353,7 +357,12 @@ namespace {
 template <typename DataManager, typename DummyUpdater, typename Generator>
 void test_data_manager_with_dummy_buffer_updater(
     const gsl::not_null<Generator*> gen,
-    const bool apply_normalization_bug = false) noexcept {
+    const bool apply_normalization_bug = false,
+    const bool is_spec_input = true) noexcept {
+  // avoid compiler warnings in the case where the normalization bug booleans
+  // aren't used.
+  (void)apply_normalization_bug;
+  (void)is_spec_input;
```

Comment:
I generally like putting these void-casts in the else branch of the constexpr, because it feels more tightly scoped and helps (me, at least) see in which branches the variables are used or not. Thoughts on this? Is there an argument I've missed for doing this at the full function scope?